### PR TITLE
[Refactor] Use MMCV MODEL_REGISTRY

### PR DESCRIPTION
**Pull request by xvjiarui**

### CODE REVIEW

File: mmdet/models/builder.py [hunk 1]
```diff
@@ -1,67 +1,47 @@
...
-        nn.Module: A built nn module.
-    """
-    if isinstance(cfg, list):
-        modules = [
-            build_from_cfg(cfg_, registry, default_args) for cfg_ in cfg
-        ]
-        return nn.Sequential(*modules)
-    else:
-        return build_from_cfg(cfg, registry, default_args)
+from mmcv.cnn import MODELS as MMCV_MODELS
```
Comment (hhaAndroid) [inline mmdet/models/builder.py:3]: Due to parameter initialization refactored, MMCV needs to modify `build_model_from_cfg`, E.g modify `nn.Sequential(*modules)` to `from ..runner import Sequential ; Sequential(*modules)`.

Comment (ZwwWayne) [reply mmdet/models/builder.py:3]: Should create a PR fast in MMCV for 1.3.2

File: mmdet/models/builder.py [hunk 2]
```diff
@@ -1,67 +1,47 @@
...
+ROI_EXTRACTORS = MODELS
+SHARED_HEADS = MODELS
+HEADS = MODELS
+LOSSES = MODELS
+DETECTORS = MODELS
 
 
 def build_backbone(cfg):
     """Build backbone."""
-    return build(cfg, BACKBONES)
```
Comment (ZwwWayne) [inline mmdet/models/builder.py:39]: Need to update the requirements of MMCV

**Comment by ZwwWayne** [comment]
This PR will be merged after the release of MMCV v1.3.2

**Review by ZwwWayne — APPROVED** [review]

**Pull request merged**